x-pack/plugins/timelines/common/alerts/constants.ts

```diff
@@ -0,0 +1,211 @@
+export const ALERT_ORIGINAL_EVENT_RISK_SCORE_NORM = `${ALERT_ORIGINAL_EVENT}.risk_score_norm` as const;
+export const ALERT_ORIGINAL_EVENT_SEQUENCE = `${ALERT_ORIGINAL_EVENT}.sequence` as const;
+export const ALERT_ORIGINAL_EVENT_SEVERITY = `${ALERT_ORIGINAL_EVENT}.severity` as const;
+export const ALERT_ORIGINAL_EVENT_START = `${ALERT_ORIGINAL_EVENT}.start` as const;
+export const ALERT_ORIGINAL_EVENT_TIMEZONE = `${ALERT_ORIGINAL_EVENT}.timezone` as const;
+export const ALERT_ORIGINAL_EVENT_TYPE = `${ALERT_ORIGINAL_EVENT}.type` as const;
+export const ALERT_ORIGINAL_EVENT_URL = `${ALERT_ORIGINAL_EVENT}.url` as const;
+export const ALERT_ORIGINAL_TIME = `${ALERT_NAMESPACE}.original_time` as const;
+export const ALERT_THREAT = `${ALERT_NAMESPACE}.threat` as const;
+export const ALERT_THREAT_FRAMEWORK = `${ALERT_THREAT}.framework` as const;
+export const ALERT_THREAT_TACTIC = `${ALERT_THREAT}.tactic` as const;
+export const ALERT_THREAT_TACTIC_ID = `${ALERT_THREAT_TACTIC}.id` as const;
+export const ALERT_THREAT_TACTIC_NAME = `${ALERT_THREAT_TACTIC}.name` as const;
+export const ALERT_THREAT_TACTIC_REFERENCE = `${ALERT_THREAT_TACTIC}.reference` as const;
+export const ALERT_THREAT_TECHNIQUE = `${ALERT_THREAT}.technique` as const;
+export const ALERT_THREAT_TECHNIQUE_ID = `${ALERT_THREAT_TECHNIQUE}.id` as const;
+export const ALERT_THREAT_TECHNIQUE_NAME = `${ALERT_THREAT_TECHNIQUE}.name` as const;
+export const ALERT_THREAT_TECHNIQUE_REFERENCE = `${ALERT_THREAT_TECHNIQUE}.reference` as const;
+export const ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE = `${ALERT_THREAT_TECHNIQUE}.subtechnique` as const;
+export const ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE_ID = `${ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE}.id` as const;
+export const ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE_NAME = `${ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE}.name` as const;
+export const ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE_REFERENCE = `${ALERT_THREAT_TECHNIQUE_SUBTECHNIQUE}.reference` as const;
+export const ALERT_THRESHOLD_RESULT = `${ALERT_NAMESPACE}.threshold_result` as const;
+export const ALERT_THRESHOLD_RESULT_CARDINALITY = `${ALERT_THRESHOLD_RESULT}.cardinality` as const;
+export const ALERT_THRESHOLD_RESULT_CARDINALITY_FIELD = `${ALERT_THRESHOLD_RESULT_CARDINALITY}.field` as const;
+export const ALERT_THRESHOLD_RESULT_CARDINALITY_VALUE = `${ALERT_THRESHOLD_RESULT_CARDINALITY}.value` as const;
+export const ALERT_THRESHOLD_RESULT_COUNT = `${ALERT_THRESHOLD_RESULT}.count` as const;
+export const ALERT_THRESHOLD_RESULT_FROM = `${ALERT_THRESHOLD_RESULT}.from` as const;
+export const ALERT_THRESHOLD_RESULT_TERMS = `${ALERT_THRESHOLD_RESULT}.terms` as const;
+export const ALERT_THRESHOLD_RESULT_TERMS_FIELD = `${ALERT_THRESHOLD_RESULT_TERMS}.field` as const;
+export const ALERT_THRESHOLD_RESULT_TERMS_VALUE = `${ALERT_THRESHOLD_RESULT_TERMS}.value` as const;
+export const ALERT_RULE_BUILDING_BLOCK_TYPE = `${ALERT_RULE_NAMESPACE}.building_block_type` as const;
+// TODO: not accounted for in the excel doc
```

Comment:
@marshallmain

Reply:
`signal.rule.exceptions_list` isn't actually mapped in the signals mapping. We'll probably want to map it in the new alerts as data schema anyway so I added it to the spreadsheet along with `signal.rule.filters`. I left `ALERT_RULE_SIZE` out for now, I'm not sure that field actually exists anywhere. Looking into it.

Reply:
Thanks!